flask_mongorest/resources.py

```diff
@@ -344,6 +357,14 @@ def value_for_field(self, obj, field):
         raise UnknownFieldError
 
     def validate_request(self, obj=None):
+        if (request.method == 'PUT' and obj) or request.method == 'POST':
```

Comment:
I thought that even bulk PUTs still call `validate_request(obj)` for each object in the bulk list.